server/Dockerfile

```diff
@@ -2,16 +2,11 @@ FROM node:10.13.0-alpine
 
 WORKDIR /usr/app
 
-COPY package.json .
-
-COPY package-lock.json .
-
-RUN npm install --quiet
-
 COPY . /usr/app
 
+RUN npm install --quiet
+RUN npm install nodemon -g
 RUN cp config-sample.json config.json
 
-RUN npx tsc
-
-CMD [ "node", "dist/index.js" ]
+CMD nodemon dist/index.js \
+ & npx tsc --watch 
```

Comment:
Before it was npx tsc building the .js file, which would then be used by the node dist/index.js, with this setup there is a watch on ts files, that after recompiling, nodemon will pick the changes up. Nodemon is just node with --watch mode on.